tests/stubs.nix

```diff
@@ -34,8 +34,8 @@ let
 
   dummyPackage = pkgs.runCommandLocal "dummy" { } defaultBuildScript;
 
-  mkStubPackage = { name ? "dummy", outPath ? null, version ? null
-    , buildScript ? defaultBuildScript }:
+  mkStubPackage' = { name ? "dummy", outPath ? null, version ? null
+    , buildScript ? defaultBuildScript, ... }:
```

Comment:
Setting it to `...` to allow any additional parameter to be passed to the package for `override` purposes.